mailpoet/tests/integration/Segments/DynamicSegments/Filters/WooCommerceCountryTest.php

```diff
@@ -136,12 +137,15 @@ private function createCustomerLookupData(array $data): void {
     $id = $connection->lastInsertId();
     $orderId = (int)$id + 1;
     $orderLookupTable = $wpdb->prefix . 'wc_order_stats';
+    $dateCreated = Carbon::now()->toDateTimeString();
     $connection->executeQuery("
-      INSERT INTO {$orderLookupTable} (order_id, status, customer_id)
+      INSERT INTO {$orderLookupTable} (order_id, status, customer_id, date_created, date_created_gmt)
```

Comment:
I wonder if we should/could use WooCommerce functions to insert data into the wp_wc_orders_stats table instead of using an insert query? I don´t think WC provides functions to manipulate wp_wc_orders_stats directly, but maybe we can achieve what we need by creating an order?

Reply:
Good point! I'll look into this. Maybe we could create an order + trigger sync to lookup tables.